airflow/sentry.py

```diff
@@ -159,8 +162,14 @@ def wrapper(task_instance, *args, **kwargs):
 
                 with sentry_sdk.push_scope():
                     try:
-                        return func(task_instance, *args, **kwargs)
+                        return func(_self, *args, **kwargs)
                     except Exception as e:
+                        # Is a LocalTaskJob
+                        if hasattr(_self, 'task_instance'):
+                            task_instance = _self
+                        else:
+                            task_instance = _self
```

Comment:
Same command in if/else?, do we even need the condition check then?

Reply:
Christ thanks for spotting this I'd some how missed the vital bit , just updated with the correct logic

Reply:
You aren't the only one to miss it!